www/js/app.js

```diff
@@ -191,10 +191,10 @@ function startSocket() {
 }
 
 function check_ping() {
-  //if ((Date.now() - last_ping) > 20000){
-  //Disable_interface(true);
-  //console.log("No heart beat for more than 20s");
-  //}
+  if ((Date.now() - last_ping) > 20000){
```

Comment:
Nice! 20 seconds seems long, should we do 10 like the old one was?

Reply:
I just used what was in the base code that was commented... I guess 10 would be ok too. the difference between this and the messaging is that this causes an action where the heartbeat you have just reports in serial...

Reply:
That makes sense